ios/Preferences.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Preferences.swift
+//  magnet
+//
+//  Created by Sam Giles on 11/7/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import Foundation
+
+class Preferences: NSObject {
+  private var db: PreferencesDB!
+  
+  override init() {
+    super.init()
+    db = PreferencesDB.get()
+  }
+  
+  func remove(prefKey: String) -> Bool {
+    return db.remove(prefKey)
```

Comment:
Regarding your question about the locking, we could start doing transactions, they run in their own queue.

Is something that I didn't do for the other sqlite implementations, but I'm planning to add:

https://github.com/stephencelis/SQLite.swift/blob/master/Documentation/Index.md#transactions-and-savepoints

More info about thread safe operations:
https://github.com/stephencelis/SQLite.swift/blob/master/Documentation/Index.md#thread-safety